### PR TITLE
Chance, "colorfulcircles"

### PR DESCRIPTION
**Pull request by MalteVoelkner**
First try on a pull request.
"colorfulcircles", Law of chance.

### CODE REVIEW

File: chance/p5js/colorfulcircles/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+function draw() {
+    
+  var x_m = width/2;
+  var y_m = height/2;
+  
+  circle_size = circle_size - (random(distance_from, distance_to));
+  var a = random(0, circle_size);
+  
+  if ( circle_size <= 0){
+    save('image.jpg');
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.js]: Please remove the save image function for the site.

File: chance/p5js/colorfulcircles/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+var transparency_to = 100;
+
+//Abstand der Kreise
+var distance_from = 0;
+var distance_to = 3;
+
+
+function setup() {   
+  createCanvas(canvas_size, canvas_size);
+  background(
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.js]: can you define a color instead making it totally random?

File: chance/p5js/colorfulcircles/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: Malte Völkner
+author-url: https://github.com/MalteVoelkner
+title: Colorfulcircles
+tags: [P5.js, chance, Law of Chance]
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.md]: If you introduce a new law you also need to write the definition for it

File: chance/p5js/colorfulcircles/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: Malte Völkner
+author-url: https://github.com/MalteVoelkner
+title: Colorfulcircles
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.md]: Two words please. Use right english grammar

File: chance/p5js/colorfulcircles/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+tags: [P5.js, chance, Law of Chance]
+permalink: /chance/colorfulcircles/
+summary: Random generation of size, color and thickness of circles
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.md]: so everything is totally random?

File: chance/p5js/colorfulcircles/index.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+---  
+
+![](path/to/another/image.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.md]: If you have no other image remove this line

File: chance/p5js/colorfulcircles/index.md (hunk 3 above)
Comment (MalteVoelkner) [inline chance/p5js/colorfulcircles/index.md]: How am I supposed to do that? In my .md file?
I can't find anything about a definition in your guide.

**Comment by MalteVoelkner** [comment]
How would I go about writing a new definition for a newly introduced law? @fabianmoronzirfas

**Comment by fabianmoronzirfas** [comment]
> How would I go about writing a new definition for a newly introduced law? @fabianmoronzirfas

Well first you should write the definition. Can be done in a gist or new repo. Then we need to identify some other sketches (or create some) where this law is applied and visible. If we are happy with the result we can add it to the site.

File: chance/p5js/colorfulcircles/index.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
+//Canvas-Größe
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.js]: English please. The whole site is in English.

File: chance/p5js/colorfulcircles/index.js [hunk 8]
```diff
@@ -0,0 +1,45 @@
+//Canvas-Größe
+var canvas_size = 500;
+
+//Startgröße Kreis
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.js]: EN please.

File: chance/p5js/colorfulcircles/index.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Malte Völkner
+author-url: https://github.com/MalteVoelkner
+title: Colorful circles
+tags: [P5.js, chance, Law of Chance]
```
Comment (fabianmoronzirfas) [inline chance/p5js/colorfulcircles/index.md]: there still is no definition for the Law of chance. If I merge this this sketch won't appear under a tag like this. Do you want to write that definition?

**Pull request merged**